src/components/NavItems/tools/SyntheticImageDetection/syntheticImageDetectionResults.jsx

```diff
@@ -312,7 +313,7 @@ const SyntheticImageDetectionResults = (props) => {
           </Grid>
           <Grid item sm={12} md={6}>
             {syntheticImageScores.length > 0 ? (
-              <Stack
+              /* <Stack
```

Comment:
You can delete the whole commented section as this is deprecated code after the refactoring